src/atc/delta/delta_handle.py

```diff
@@ -110,6 +112,35 @@ def recreate_hive_table(self):
         self.drop()
         self.create_hive_table()
 
+    def get_partitioning(self):
+        """The result of DESCRIBE TABLE tablename is like this:
+        +-----------------+---------------+-------+
+        |         col_name|      data_type|comment|
+        +-----------------+---------------+-------+
+        |           mycolA|         string|       |
+        |           myColB|            int|       |
+        |                 |               |       |
+        |   # Partitioning|               |       |
+        |           Part 0|         mycolA|       |
+        +-----------------+---------------+-------+
+        but thie method return the partitioning in the form ['mycolA']
+        """
+        if self._partitioning is None:
+            rows_iter = iter(
+                Spark.get().sql(f"DESCRIBE TABLE {self.get_tablename()}").collect()
+            )
+            for row in rows_iter:
```

Comment:
could you provide a # comment on what this logic does?

Reply:
added comments.